tests/test_param_validation.py

```
@@ -0,0 +1,254 @@
+"""Tests for parameter validation using Pydantic models."""
+
+import pytest
+from pydantic import ValidationError
+
+from docker_mcp.models.params import (
+    DockerComposeParams,
+    DockerContainerParams, 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)